gdx/src/com/badlogic/gdx/utils/Json.java

```diff
@@ -76,6 +77,10 @@ public void setIgnoreUnknownFields (boolean ignoreUnknownFields) {
 		this.ignoreUnknownFields = ignoreUnknownFields;
 	}
 
+	public boolean isIgnoreUnknownFields () {
```

Comment:
Let's use `getIgnoreUnknownFields`. Using `is` for a prefix is often awkward, then we end up with a mix of `get` and `is` for boolean properties.